identity/webapp/pages/registration.tsx

```diff
@@ -47,8 +48,10 @@ export const getServerSideProps: GetServerSideProps<Props | AppErrorProps> =
     );
     let email = '';
 
+    const config = getConfig();
```

Comment:
Ah yeah, I think this was because I wasn't mocking `next/getConfig` in the new registration tests – I got some sort of error from TypeScript, but if I move it back out now it seems fine.